### PR TITLE
Physionet

### PR DESCRIPTION
**Pull request by bpark738**
LstmSingleTask.java

### CODE REVIEW

File: training-labs/src/main/java/ai/skymind/training/solutions/LstmSingleTask.java [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+import org.nd4j.linalg.dataset.api.iterator.DataSetIterator;
+import org.datavec.api.transform.transform.column.RemoveColumnsTransform;
+
+/**
+ * EXERCISE 4: train a LSTM to predict mortality using the Physionet
+ * Challenge 2012 data publicly available at https://physionet.org/challenge/2012/
+ *
+ *
+ */
+public class LstmSingleTask {
```
Comment (turambar) [inline training-labs/src/main/java/ai/skymind/training/solutions/LstmSingleTask.java]: Rename class (and file) to PhysionetMultivariateTimeSeriesClassification.java. Assume @tomthetrainer approves that name. ;)

**Review by turambar — CHANGES_REQUESTED** [review]
Just one minor change.

**Comment by bpark738** [comment]
Renamed file and updated class name

**Comment by tomthetrainer** [comment]
This needs to change to ClassPathResource and point to resources content. Or needs to be automated and described in a setup document

private static File baseDir = new File("/Users/Briton/Desktop/resourcesP/physionet2012");
    private static File featuresDir = new File(baseDir, "sequence");

**Comment by tomthetrainer** [comment]
if class name changes then logger name changes,
private static final Logger log = LoggerFactory.getLogger(PhysionetMultivariateTimeSeriesClassification.class);

**Comment by tomthetrainer** [comment]
Minor issues, 
The snapshot requirement, I defer to Dave and Britons opinion on that, Any change like that to the POM and I have to run all examples and verify they still work, if things do not work then life is hard. Changing code is also hard, so do what you think is right. 

Logging: 
I changed this line..

// For logging with SL4J
    private static final Logger log = LoggerFactory.getLogger(PhysionetMultivariateTimeSeriesClassification.class);

I also had to add this because I do not have Log4J configuration in my path or something. 

public static void main(String[] args) throws IOException, InterruptedException {
        **BasicConfigurator.configure();**

Also we have two resources folders, we should use just the one. 
class_labs/src/main/resources
and
class_labs/training-labs/src/main/resources

Move physionet from 
class_labs/src/main/resources
to
class_labs/training-labs/src/main/resources

**Pull request merged**